v2/emailpassword/common-customizations/signin-form/customising-each-form-field.mdx

```diff
@@ -0,0 +1,210 @@
+---
+id: customising-each-form-field
+title: Customising each form field
+hide_title: true
+show_ui_switcher: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./emailpassword/common-customizations/signin-form/customising-each-form-field.mdx -->
+
+import TabItem from '@theme/TabItem'
+import {PreBuiltOrCustomUISwitcher, PreBuiltUIContent, CustomUIContent} from "/src/components/preBuiltOrCustomUISwitcher"
+import FrontendPreBuiltUITabs from "/src/components/tabs/FrontendPreBuiltUITabs";
+
+
+# Customising each form field
+
+* [Modify Labels and Placeholder Text](#modify-labels-and-placeholder-text)
+* [Setting Default Values](#setting-default-values)
+* [Changing Optional Error Message](#changing-optional-error-message)
+
+
+<PreBuiltOrCustomUISwitcher>
+
+## Modify Labels and Placeholder Text
+
+<PreBuiltUIContent>
+
+If you would like to modify the fields in the login widget, by changing UI labels or placeholder text, you can do so by modifying the `formFields` property when initializing SuperTokens on the frontend.
+
+<FrontendPreBuiltUITabs>
+<TabItem value="reactjs">
+
+```tsx
+import SuperTokens from "supertokens-auth-react";
+import ^{recipeNameCapitalLetters} from "supertokens-auth-react/recipe/^{codeImportRecipeName}";
+import Session from "supertokens-auth-react/recipe/session";
+
+SuperTokens.init({
+    appInfo: {
+        apiDomain: "...",
+        appName: "...",
+        websiteDomain: "..."
+    },
+    recipeList: [
+         ^{recipeNameCapitalLetters}.init({
+            signInAndUpFeature: {
+                signInForm: {
+                    // highlight-start
+                    formFields: [{
+                        id: "email",
+                        label: "customFieldName",
+                        placeholder: "Custom value"
+                    }]
+                }
+                // highlight-end
+            }
+        }),
+        Session.init()
+    ]
+});
+```
+</TabItem>
+</FrontendPreBuiltUITabs>
+
+<img alt="Prebuilt form UI with custom labels and placeholder" src="/img/emailpassword/modified-formfields.png" />
```

Comment:
this image in the doc is MASSIVE! Set proper height or width here to make it look proper.. applies to all images.